src/Aspire.Dashboard/Components/Controls/FluentIconSwitch.razor

```diff
@@ -0,0 +1,36 @@
+﻿<FluentButton Appearance="@Appearance"
+              Disabled="Disabled"
+              Title="@(Value is true ? CheckedTitle : UncheckedTitle)"
+              aria-label="@(Value is true ? CheckedTitle : UncheckedTitle)"
+              OnClick="@(() => OnToggleInternalAsync())"
+              IconEnd="@(Value is true ? CheckedIcon : UncheckedIcon)"
+              slot="end" />
```

Comment:
`slot` shouldn't be part of the component. It should be an attribute you can add to the component and is passed to the fluent button.

Reply:
Something like this:

* https://github.com/microsoft/fluentui-blazor/blob/ef53ebde9544fd1b58b8ceca6744d1860f459285/src/Core/Components/Base/FluentComponentBase.cs#L54-L58
* https://github.com/microsoft/fluentui-blazor/blob/ef53ebde9544fd1b58b8ceca6744d1860f459285/src/Core/Components/Button/FluentButton.razor#L47

Reply:
This has been addressed